cmd/env_test.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+	"github.com/spf13/viper"
+)
+
+func TestEnv(t *testing.T) {
+	Convey("env", t, func() {
+		viper.Reset()
+
+		Convey("returns env for a valid environment", func() {
+			viper.Set("environments", map[string]interface{}{
+				"edge-cmh": "kubeconfig-truss-nonprod-cmh",
+			})
+
+			cmd := rootCmd
+			buff := bytes.NewBufferString("")
+			cmd.SetOut(buff)
+			cmd.SetArgs([]string{
+				"env",
+				"-e",
+				"edge-cmh",
+			})
+			cmd.Execute()
+			out, _ := ioutil.ReadAll(buff)
+			So(string(out), ShouldEqual, "export KUBECONFIG=/Users/jblanchard/.kube/kubeconfig-truss-nonprod-cmh\n# Run this command to configure your shell:\n# eval \"$(truss env -e edge-cmh)\"\n")
```

Comment:
can we stub something so this isn't using your homedir?

Reply:
Ah yeah, good call. I'll make it configurable or just check for a substring or something.